lib/routes/account.js

```diff
@@ -96,7 +96,8 @@ module.exports = function (
         var authPW = Buffer(form.authPW, 'hex')
         var locale = request.app.acceptLanguage
         var userAgentString = request.headers['user-agent']
-        var service = form.service || request.query.service
+        var query = request.query
```

Comment:
tiny nit: for organizational purposes, I think this should live right next to the line that does `form = request.payload`.